.tx/config

```diff
@@ -12,3 +12,4 @@ file_filter = client/lang/src/<lang>.js
 source_file = client/lang/src/en.js
```

Comment:
Please remove the js config from the `.tx` since these are removed from framework.